uqcsbot/utils/snailrace_utils.py

```diff
@@ -17,7 +17,11 @@
 SNAILRACE_MAX_STEP = 15
 SNAILRACE_MIN_STEP = 2
 
+# snailrace channels
+BLESSED_CHANNELS = ["general", "bot-testing", "banter", "games", "snail-race"]
```

Comment:
following a brief consultation with committee, they'd like to be limited to just #bot-testing to start with. if/when more games might get added to the bot (no comment) they'll look at making a dedicated "bot games" channel.